container/health_check.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "hello-openshift4",
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "hello-openshift",
+        "labels": {
+            "name": "hello-openshift"
+        }
+    },
+    "spec": {
+        "containers": [
+            {
+                "name": "hello-openshift",
+                "image": "zzhao/hello-openshift",
+                "ports": [
+                    {
+                        "hostPort": 6064,
```

Comment:
if you do not need web access, you can delete the hostpord for your pod, since it needs change SCC to make your pod valid to be created.